tools/export/cdt/__init__.py

```diff
@@ -12,13 +14,14 @@ def generate(self):
         py_ocd_settings launch file, and software link .p2f file
         """
         super(Eclipse, self).generate()
+        include_paths_replace_re= re.compile(r'(^[.]/|^[.]$)')
```

Comment:
Could this variable have a shorter name? maybe `starting_dot`? which implies that this is the same as `<string>.startswith(".")`